src/main/java/org/skife/jdbi/v2/sqlobject/BindFactory.java

```diff
@@ -13,16 +13,13 @@
  */
 package org.skife.jdbi.v2.sqlobject;
 
-import java.lang.annotation.Annotation;
-
-class BindFactory implements BinderFactory
+class BindFactory implements BinderFactory<Bind>
 {
     @Override
-    public Binder build(Annotation annotation)
+    public Binder build(Bind bindAnnotation)
     {
-        Bind bind = (Bind) annotation;
         try {
-            return bind.binder().newInstance();
+            return bindAnnotation.binder().newInstance();
```

Comment:
Could we just rename the parameter to `bind` like the local variable, to reduce LOC changes?